### ISSUE TITLE
Weird thing about Cookie Clicker

### ISSUE THREAD

**Issue by Matocolotoe** [NiklasEi/gamebox#62]
Are `data.yml` and `saves.yml` synchronized regarding the amount of total cookies per player ?
Because when the server reboots and some of the cookies haven't been saved, the total amount of cookies in the top list increases but not for the player

**Comment by NiklasEi** [comment]
The cookie clicker gui will only tell you how many cookies you own at the moment. The top list tells you how many you produced.

So if you produce 15 cookies, then buy a cursor for 10, you own 5, but your highscore is 15.

**Comment by Matocolotoe** [comment]
ikr, but a player had like 30 million cookies, the server rebooted, the top list score and his personal total score had a difference of about 2 million cookies

**Comment by NiklasEi** [comment]
Two million cookies is not much. There are a lot of updates or buildings that cost a lot more than that. Are you sure that he didn't just buy something?
I am not saying, that I am 100% sure that nothing went wrong, but for now, it doesn't sound like it. Were there any errors in the console? Does this happen regularly?

**Comment by Matocolotoe** [comment]
No, it happens when the server reboots while people are playing Cookie Clicker
But i'm pretty sure there was a difference between the score in the top list and the personal total amount
